app/javascript/components/project-submissions/components/__tests__/modal.test.js

```diff
@@ -0,0 +1,48 @@
+import { fireEvent, render } from "@testing-library/react";
+import React from "react";
+import Modal from "../modal";
+
+describe("Modal", () => {
+  test("Checks for the specified classname on modal when the modal is not shown", () => {
+    const { getByLabelText } = render(
+      <Modal 
+        handleClose={() => {}} 
+        show={false}
+      >
+        {" "}
+      </Modal>
+    );
+
+    expect(getByLabelText("modal--hidden")).toHaveClass("react-modal--hidden");
```

Comment:
Test should not rely solely on an assertion on a class name, which is opaque and unnecessary to a user. Add an assertion that tests for the presence/absence of children, per line 14 of the component. You may want to pass in a simple `div` with some text to assert against.